core/templates/pages/skill-editor-page/services/skill-editor-state.service.ts

```diff
@@ -59,7 +59,7 @@ export class SkillEditorStateService {
   private _skill!: Skill;
   private _skillRights!: SkillRights;
   private _skillIsInitialized: boolean = false;
-  private assignedSkillTopicData = null;
+  private assignedSkillTopicData: { [topicName: string]: string } | null = null;
```

Comment:
Can you put this type into a type variable?

Reply:
Will do! Could you please suggest an appropriate line to declare a new type?

Reply:
Somewhere in the top of the file

Reply:
Done!